src/devtools/client/debugger/src/actions/pause/selectFrame.ts

```diff
@@ -32,7 +33,16 @@ export function selectFrame(cx: Context, frame: TempFrame): UIThunkAction {
 
     dispatch(frameSelected({ cx, frameId: frame.id }));
 
-    await ThreadFront.getFrameSteps(frame.asyncIndex, frame.protocolId);
+    try {
+      await ThreadFront.getFrameSteps(frame.asyncIndex, frame.protocolId);
+    } catch (e: any) {
+      // TODO [FE-795]: Communicate this to the user
+      if (e.code === ProtocolError.TooManyPoints) {
+        console.error(e);
```

Comment:
Oh, no particular reason, I was just shuffling the original code around. I will add this elsewhere, it seems like a helpful thing to have.

Reply:
Sure, agreed! I should have been more clear: We should probably _also_ log errors in `setFramePositions` since otherwise they would just fail silently.

Reply:
Done!